specs/linux/startup_items.table

```diff
@@ -0,0 +1,13 @@
+table_name("startup_items")
```

Comment:
This is a duplicate of the existing `startup_items.table`, we don't need this new file.